### PR TITLE
Update description get started

### LINKED ISSUE #52: Invalid description in the get started
The second text of the get started is wrong, it's asking to run a command where it should ask to create a file

### PR DESCRIPTION
**Pull request by antho1404**
fix #52

### CODE REVIEW

**Review by EmmanuelDrouin — CHANGES_REQUESTED** [review]
Great it's way better. I would like to precise that the files should be created at the root of the directory (myfunction.js and process). Cause it's something i see all the time in the feedback, users ask themself the question "where should i had/create this file?"

File: content/en/index.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ npm install -g @liteflow/cli
 
 ## 2. Create your function
 
-In order to create an application, run the following command:
+Let's create our function that will accept the request's inputs and return an object with `greetings` based on the input of the request.
+
+Start by creating a new file called `myfunction.js` with the following content:
```
Comment (antho1404) [inline content/en/index.md:26]:
```suggestion
Start by creating a new file at the root of your project directory called `myfunction.js` with the following content:
```

File: content/en/index.md [hunk 2]
```diff
@@ -34,7 +36,9 @@ module.exports = (inputs) => {
 
 ## 3. Connect your function to an HTTP endpoint
 
-Create a new [process](/process) that creates an HTTP endpoint and connect each requests on this endpoint to your function.
+Now that our function is defined, let's connect it to an [HTTP endpoint](/services/http#request) through a [process](/process).
+
+Create a new file `process.yml` with an endpoint trigger and a task that uses the defined function.
```
Comment (antho1404) [inline content/en/index.md:41]:
```suggestion
Create a new file at the root of your project directory called `process.yml` with an endpoint trigger and a task that uses the defined function.
```

**Comment by antho1404** [comment]
I did some updates

**Review by EmmanuelDrouin — APPROVED** [review]
Great perfect! Let's merge

**Pull request merged**